app/models/manageiq/providers/container_manager.rb

```diff
@@ -37,5 +37,13 @@ def aggregate_memory(targets = nil)
     def chart_layout_path
       "ManageIQ_Providers_ContainerManager"
     end
+
+    def self.cve_check
+      # TODO: optimize by keeping last hashes of cve streams raising only upon update
+      MiqServer.my_server.zone.ext_management_systems.each do |ems|
+        first_image = ems.container_images.first
```

Comment:
The `ems` variable is not going to be limited to just container_manager providers which means the `ems.container_images.first` call is going to fail.

Was there an issue with the suggested code change I recommended?